### DISCUSSION TITLE
Under Championship Predictions, NaN

### DISCUSSION THREAD

**Discussion by rndtan** [f1multiviewer/issue-tracker#370] [Feature requests] [answered]
Within the Live Timing winder, clicking on the trophy icon reveals a modal. Within modal, the left column displays +NaN for multiple drivers (Leclerc, Ocon, Piastri)

![Screenshot 2023-03-19 at 13 03 18](https://user-images.githubusercontent.com/751510/226198623-03a5439a-369f-4df1-afa7-4357c65db90b.png)

**Comment by f1multiviewer** [comment, marked as answer]
This will be fixed in the next release
